commit 98ab29f9cc1261e9eeda97621924ee3de7302e94 (refs/changes/54/4154/1)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-06-05T13:28:51-04:00 (1 year, 4 months ago)
    
    TRITON-461 triton services still on nodejs 0.10.x should at least use the latest x

diff --git a/Makefile b/Makefile
index c06c4d5..135553e 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright 2016, Joyent, Inc.
+# Copyright 2018, Joyent, Inc.
 #
 
 #
@@ -40,7 +40,7 @@ REPO_MODULES	 = src/node-dummy
 SMF_MANIFESTS_IN = smf/manifests/firewaller.xml.in
 
 ifeq ($(shell uname -s),SunOS)
-	NODE_PREBUILT_VERSION=v0.10.26
+	NODE_PREBUILT_VERSION=v0.10.48
 	NODE_PREBUILT_TAG=gz
 	NODE_PREBUILT_IMAGE=fd2cc906-8938-11e3-beab-4359c665ac99
 endif
